src/components/MultiSelect/index.js

```diff
@@ -181,20 +199,20 @@ const MultiSelect = React.forwardRef((props, ref) => {
                     <RenderIf isTrue={!shouldRenderChips}>
                         <StyledPlaceholder>{placeholder}</StyledPlaceholder>
                     </RenderIf>
-                    <RenderIf isTrue={shouldRenderChips}>
-                        <Chips value={value} variant={chipVariant} onDelete={handleDelete} />
-                    </RenderIf>
+                    <RenderIf isTrue={shouldRenderChips}>{content}</RenderIf>
```

Comment:
I think I should rethink here because it works but it is difficult to understand. Something like that would be easier to understand.

```
const content = variant === 'default' && getContent(value);
const chips = variant === 'chip' && hasChips(value);
```

```
<RenderIf isTrue={!content && !chips}>
    <StyledPlaceholder>{placeholder}</StyledPlaceholder>
</RenderIf>
<RenderIf isTrue={!!chips}>
    <Chips value={value} variant={chipVariant} onDelete={handleDelete} />
</RenderIf>

<RenderIf isTrue={!!content}><StyledText>{content}</StyledText></RenderIf>
```

Reply:
we should be careful here since `!content && !chips` is hard to read because of the negates

Reply:
also, the use of const like shouldRenderChips indicates explicitly what is the intention

Reply:
what can be done is change shouldRenderChips to shouldRenderContent or something like that since there is not only chips now

Reply:
Done, renamed shouldRenderChips to shouldRenderContent, also moved all other RenderIf conditions to shouldRender* const variables